README.md

```diff
@@ -29,31 +29,13 @@ Such a procedure is [described in the c:geo wiki](https://github.com/cgeo/cgeo/w
 
 An overview of released beta and production versions and their changelog can be found on the [release page of this repository](https://github.com/cgeo/WhereYouGo/releases).
 
-# Old README.md
```

Comment:
If you remove the "Old README" section then you should change in all sections :-)

Reply:
Oh yes. Too tired. Will continue tomorrow.